client/dive-common/components/Viewer.vue

```diff
@@ -206,6 +200,21 @@ export default defineComponent({
       removeTrack,
     });
 
+    const allSelectedIds = computed(() => {
+      if (selectedTrackId.value) {
```

Comment:
Missed it before, but caught it now.  `selectedTrackId = 0` means a false here.  Needs to be `!== null`?

Reply:
:skull_and_crossbones: